.travis.yml

```diff
@@ -2,7 +2,7 @@ language: python
 python:
   - "2.7"
 install:
-  - pip install ansible==2.4.2
+  - pip install ansible==2.8.3
```

Comment:
I see in the README.md that we're forcing `dnspython==1.15.0`. Do we want to do that also here?

Reply:
That is probably a good idea too.